snforge_std/src/cheatcodes.cairo

```diff
@@ -26,10 +26,10 @@ fn test_address() -> ContractAddress {
 }
 
 /// Changes the block number for the given target and span.
-/// - `target` - instance of `ContractAddress` specifying which contracts to roll
+/// - `target` - instance of `ContractAddress` specifying which contracts to cheat_block_number
```

Comment:
These comments are going to need some rewording if we want them to keep some sense